### PR TITLE
Add roman-numerals

### PR DESCRIPTION
**Pull request by BNAndras**
@ErikSchierboom, one for your review.

Related to #236

### CODE REVIEW

**Review by ErikSchierboom — COMMENTED** [review]
Looks good, but CI is failing

**Comment by BNAndras** [comment]
CI fixed

File: exercises/practice/roman-numerals/roman-numerals.coffee
Comment (ErikSchierboom) [inline exercises/practice/roman-numerals/roman-numerals.coffee:1]: Example solution is in the stub :)

Comment (BNAndras) [reply exercises/practice/roman-numerals/roman-numerals.coffee:1]: Whoops, thanks. Might be time to go to bed :)

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**